tests/provision/facts/test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "run=\$(mktemp -d)" 0 "Create run directory"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Check $provision_method plugin"
+        if [ -f /etc/os-release ]; then
+            distro="$(grep PRETTY_NAME /etc/os-release | cut -d'=' -f2 | tr -d '"')"
+        else
+            distro="$(cat /etc/redhat-release || cat /etc/fedora-release)"
+        fi
+
+        if [ grep "selinuxfs" /proc/filesystems ]; then
+            selinux="no"
+        else
+            selinux="yes"
+        fi
+
+        rlRun -s "tmt run -i $run --scratch     provision -h local plan -n /plans/features/core"
```

Comment:
Command uses `provision -h local` but the test phase name hints ` $provision_method`.
IMO we should test more methods (the way of checking might be different) so could you use the parameter here?

Reply:
Fixed in 76a8f4d3